README.md

```diff
@@ -55,42 +63,42 @@ git submodule update --init --recursive
 
 ## 2. Get the Docker images with `=nil;` toolchain
 
-We've prepared Docker images with parts of the `=nil;` toolchain 
-that you'll need to run through all development stages with this template.
+In the tutorial we will use Docker images with parts of the `=nil;` toolchain.
 We recommend using them because they're tested for compatibility,
-and they save you time on installing and compiling everything.
+and they save you time on installing and compiling everything:
 
-* `nilfoundation/zkllvm-template` has the zkLLVM part of the toolchain:
+* The `nilfoundation/zkllvm-template` image has the zkLLVM part of the toolchain:
```

Comment:
That's the second colon here (first on line 68), maybe replace it with a dash or comma?

Reply:
fixed